pkg/packagekit/updates.jsx

```diff
@@ -142,6 +143,14 @@ function count_security_updates(updates) {
     return num_security;
 }
 
+function find_highest_severity(updates) {
+    var max = PK.Enum.INFO_LOW;
+    for (const u in updates)
+        if (updates[u].severity > max)
+            max = updates[u].severity;
+    return max;
+}
```

Comment:
This would fit well right next to `getSeverityIcon()` in pkg/lib/packagekit.js instead.

Reply:
The elements of `updates` are custom structs created in this file, so I don't think lib/packagekit.js should know about them.

Reply:
Ack.